src/deepsparse/server/config.py

```diff
@@ -72,6 +111,10 @@ class EndpointConfig(BaseModel):
         default=1, description="The batch size to compile the model for."
     )
 
+    data_logging: Optional[
+        List[Dict[str, Union[List[MetricFunctionConfig], str]]]
```

Comment:
per @corey-nm 's catch, we don't need this outer list, each identifier should be able to map to a list of its functions like so
```yaml
data_logging:
  target_1:
     - function: any
       freq: 1
     - function: any
       freq
  target_2: [{...}, ...]
```

```suggestion
        Dict[str, List[MetricFunctionConfig]]
```